src/dispatch/static/dispatch/src/incident/ResourcesTab.vue

```diff
@@ -62,23 +62,83 @@
       </v-list-item>
       <v-divider />
     </span>
+    <span
+      v-if="
+        (!ticket && ticketPluginEnabled) ||
+        (!conference && conferencePluginEnabled) ||
+        (!conversation && conversationPluginEnabled) ||
+        (!storage && storagePluginEnabled) ||
+        (!documents && documentPluginEnabled)
```

Comment:
I think we're missing groups in this component.

Reply:
I chose not to include groups in this check since groups data was not rendered in this sheet.

Reply:
I'll resolve this in a separate PR.